RELEASING.md

```diff
@@ -25,7 +26,7 @@ Don't be scared off by this document! If you already understand the stuff in th
 <details>
 <summary>Eclipse OpenJ9/OMR releases</summary>
 
-### NOTE: Temurin does not ship OpenJ9 releases so this section has been collapsed as it it not required by anyone performing the Temurin release process
+### NOTE1: Temurin does not ship OpenJ9 releases so this section has been collapsed as it it not required by anyone performing the Temurin release process
```

Comment:
Note sure about `NOTE1`?  Is that specific Markdown syntax?

Reply:
that is to fix the linter error, because we have two places in the same md file with exactly the same header line.
Open for suggestion :)

Reply:
Ah all good, thanks!